docker-compose.yml

```diff
@@ -134,7 +134,7 @@ services:
       - SCREENSHOT_BUCKET=email-screenshots
       - AWS_REGION=us-east-1
       - DISABLE_DD=true
-    container_name: screenshot-service
+    container_name: sublime_screenshot-service
```

Comment:
This container is referenced on L14 and I believe the name needs to be updated. I think this might apply for a few of these containers.

Reply:
I thought that references are based on service name not container name?

Reply:
Ooh ya I think you're right, disregard!

Looks like we have an existing usage that shows this: `strelka_frontend` is addressed by the service name on L13 (it already has a different container name).